test/controllers/TransferSummaryControllerSpec.scala

```diff
@@ -151,11 +151,83 @@ class TransferSummaryControllerSpec extends FrontEndTestHelper {
+      print(contentAsString(finalTransferConfirmationSubmitResult))
+      contentAsString(finalTransferConfirmationSubmitResult) must include("govuk-error-message")
+      contentAsString(finalTransferConfirmationSubmitResult) must include("error")
+      contentAsString(finalTransferConfirmationSubmitResult) must include("There is a problem")
+      contentAsString(finalTransferConfirmationSubmitResult) must include("#error-transferLegalOwnership")
+      contentAsString(finalTransferConfirmationSubmitResult) must include("transferSummary.transferLegalOwnership.error")
+    }
+
+    "display correct error when only the transfer legal ownership option is selected and the final transfer confirmation form is submitted" in {
+      val client = new GraphQLConfiguration(app.configuration).getClient[gcs.Data, gcs.Variables]()
+      val seriesCode = Some(gcs.GetConsignment.Series(Some("Mock Series 2")))
+      val transferringBodyName = Some(gcs.GetConsignment.TransferringBody(Some("MockBody 2")))
+      val totalFiles: Int = 4
+      val consignmentReference = Option("TEST-TDR-2021-GB")
+
+      val consignmentResponse: gcs.GetConsignment = new gcs.GetConsignment(seriesCode, transferringBodyName, totalFiles, consignmentReference)
+      val data: client.GraphqlData = client.GraphqlData(Some(gcs.Data(Some(consignmentResponse))), List())
+      val dataString: String = data.asJson.printWith(Printer(dropNullValues = false, ""))
+      wiremockServer.stubFor(post(urlEqualTo("/graphql"))
+        .willReturn(okJson(dataString)))
+
+      val graphQLConfiguration = new GraphQLConfiguration(app.configuration)
+      val consignmentService = new ConsignmentService(graphQLConfiguration)
+
+      val controller = new TransferSummaryController(getAuthorisedSecurityComponents, new GraphQLConfiguration(app.configuration),
+        getValidKeycloakConfiguration, consignmentService, exportService(app.configuration), langs)
+      val finalTransferConfirmationSubmitResult = controller.finalTransferConfirmationSubmit(consignmentId)
+        .apply(FakeRequest(POST, s"/consignment/$consignmentId/transfer-summary")
+          .withFormUrlEncodedBody(finalTransferConfirmationForm(openRecordsValue = false, transferLegalOwnershipValue = true): _*)
+          .withCSRFToken)
+
+      playStatus(finalTransferConfirmationSubmitResult) mustBe BAD_REQUEST
+      print(contentAsString(finalTransferConfirmationSubmitResult))
+
       contentAsString(finalTransferConfirmationSubmitResult) must include("govuk-error-message")
       contentAsString(finalTransferConfirmationSubmitResult) must include("error")
+      contentAsString(finalTransferConfirmationSubmitResult) must include("There is a problem")
+      contentAsString(finalTransferConfirmationSubmitResult) must include("#error-openRecords")
+      contentAsString(finalTransferConfirmationSubmitResult) must include("transferSummary.openRecords.error")
```

Comment:
Might be worth adding another check to make sure the other error is not displayed.